javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/OpenTelemetrySdkAccess.java

```diff
@@ -22,13 +22,13 @@ public final class OpenTelemetrySdkAccess {
    */
   public interface ForceFlusher {
     /** Executes force flush. */
-    void run(int timeout, TimeUnit unit);
+    void run(long timeout, TimeUnit unit);
   }
 
   private static volatile ForceFlusher forceFlush;
 
   /** Forces flushing of pending spans. */
-  public static void forceFlush(int timeout, TimeUnit unit) {
+  public static void forceFlush(long timeout, TimeUnit unit) {
```

Comment:
maybe keep and mark the `int` variant deprecated for removal in 3.0? seems like something that _could_ be in use by an extension

Reply:
I added back the original method. If the indy extension changes get done in time for 3.0 it is possible that we won't need most of the bootstrap classes any more.